backend/app.js

```diff
@@ -1,15 +1,33 @@
 const { engine } = require("express-handlebars");
 const express = require("express");
+const cors = require("cors");
 const { sendOrderConfirmationEmail } = require("./email");
+const { login, isAuthenticated } = require("./utils/login");
 
 const app = express();
 app.engine("handlebars", engine());
 app.set("view engine", "handlebars");
+app.use(cors());
 app.set("views", "./views");
 const PORT = process.env.PORT || 3030;
 
 app.use(express.json());
 
+app.post("/admin/login", async (req, res) => {
+  let userCredentials = req.body;
+  const authUser = login(userCredentials);
+  if (authUser) {
+    res.send({ token: authUser });
+  } else {
+    res.send({ error: "invalid credential" });
+  }
+});
+app.post("/admin/isAuthenticated", async (req, res) => {
+  console.log("called");
+  const userToken = req.body.token;
+  res.send({ isAuthenticated: isAuthenticated(userToken) });
+});
```

Comment:
I think you could use try/catch and add status to your res for better representation of the response being send.